src-electron/generator/matter/app/zap-templates/common/ChipTypesHelper.js

```diff
@@ -54,5 +54,5 @@ function asBasicType(type) {
 exports.asBasicType = asBasicType;
 exports.meta = {
   category: dbEnum.helperCategory.matter,
-  alias: ['common/ChipTypesHelper.js'],
+  alias: ['common/ChipTypesHelper.js', 'matter-chip-types-helper'],
```

Comment:
accident w/ merge?

Reply:
After rebasing that with master this was appeared I think